docs/api_reference/Models/KeyResponse.md

```diff
@@ -1,10 +1,11 @@
-# PublicKeyResponse
+# KeyResponse
 ## Properties
 
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 **kid** | **String** |  | [default to null]
-**public\_key** | **String** |  | [default to null]
+**status** | **String** |  | [default to null]
+**key** | **String** |  | [default to null]
```

Comment:
Q: As key in response is base64 encoded, should we return the `format` as a separate field (especially in case we default to a format) to avoid confusion?